pkg/frontend/adminactions/resources_list_test.go

```diff
@@ -91,12 +91,17 @@ func TestResourcesList(t *testing.T) {
 						ProvisioningState: to.StringPtr("Succeeded"),
 					},
 				}, nil)
+
+				diskEncryptionSets.EXPECT().Get(gomock.Any(), gomock.Any(), gomock.Any()).Return(mgmtcompute.DiskEncryptionSet{
+					ID:   to.StringPtr("/subscriptions/id"),
+					Type: to.StringPtr("Microsoft.Compute/diskEncryptionSets"),
+				}, nil)
```

Comment:
We need to cover multiple cases:

* No disk encryption set in the cluster document (we no longer covering it becasue of this change)
* Disk encryption set is present in the cluster document, all good.
* Disk encryption set is present in the cluster document, but get `a.diskEncryptionSet.Get` returns an error. What happens in this case? Do we return an error in geneva action or do we silently return a result?

Given the above, I think "basic coverage" test case should stay basic: no disk encryption set. And we need to add few more new test cases.